backend/remote-state/swift/client.go

```diff
@@ -63,23 +87,134 @@ func (c *RemoteClient) ListObjectsNames(prefix string, delim string) ([]string,
 
 }
 
-func (c *RemoteClient) Get() (*remote.Payload, error) {
-	log.Printf("[DEBUG] Getting object %s in container %s", c.objectName, c.container)
-	if err := c.ensureContainerExists(); err != nil {
-		return nil, err
+// GetNewestOpts add a parameter to ensure the objects returned are the most
+// recent known by swift.
+// swift is eventually consistent. Yet end users may expect to get a consistent
+// response when querying for state or lock objects
+type GetNewestOpts struct{}
```

Comment:
@yanndegat 

I've just merged support for `Newest` to Gophercloud: https://github.com/gophercloud/gophercloud/issues/1118

You should be able to do a vendor update and then not have to create a custom type.